sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/implementation/signature/KeyVaultKeyLessRsaSignature.java

```diff
@@ -54,4 +55,14 @@ protected void engineSetParameter(AlgorithmParameterSpec params)
     public String getAlgorithmName() {
         return "RSASSA-PSS";
     }
+
+    /**
+     * add this method to enable engineGetParameters which added in this commit:
+     * https://github.com/openjdk/jdk/commit/316140ff92af7ac1aadb74de9cd37a5f3c412406
```

Comment:
I can't find `engineGetParameters` in this commit , did I missed something?

Reply:
My fault, in fact, it added the logic of `signer.getParameters()`, which you can see here: https://github.com/openjdk/jdk/commit/316140ff92af7ac1aadb74de9cd37a5f3c412406#diff-15bf9a7f6d17383a3e3c8aa69ce0dd21757e64fe4194bd931978d45ad1a00127 and line 202. Then the engineGetParameters() method is called in this method(`getParameters()`). Maybe I can add more detailed instructions?

Reply:
Do we need to provide all `SignatureSpi`'s methods for all our Signature class instead of implement these method when it throw exception?

Reply:
I think this is a good idea, but in fact we have implemented most of the methods, and I don't think we need to implement the remaining methods, because they have already been done in `SignatureSpi`. But I do think that `engineGetParameters` should be implemented in `AbstractKeyVaultKeyLessSignature` to ensure that our other signatures can also support this method, just like `engineSetParameter`. I will try to move this function and check if it can work normally.
![image](https://user-images.githubusercontent.com/70368631/129656030-1607d7f2-c4a1-49c6-bdbd-e399da92db82.png)